pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1552,4 +1552,7 @@ self: super: {
   servant-client      = doJailbreak super.servant-client;
   servant-client-core = doJailbreak super.servant-client-core;
 
+  # overly strict dependency on aeson
+  profiteur = doJailbreak (super.profiteur);
```

Comment:
Is there an upstream issue or pull request which would fix the issue? If yes, it would be nice to link it in the comment here, so it's easy to see when this line can be removed again. If no, it would be great if you could open an issue :)

Reply:
I'm sorry @cleverca22, I should have pointed this out in my review yesterday as well.

If there is an upstream issue/pr for this, please link it here in a comment.  If not, please create one and link it here.

This is so that the nixpkgs maintainers can track when these uses of `doJailbreak` are no longer necessary.

Reply:
https://github.com/jaspervdj/profiteur/issues/33
the gentoo folks beat me to it!